README.md

```diff
@@ -1,70 +1,222 @@
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- Have a local version control like Git. Which is a open source distributed version control system designed for source code management
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content (PHP, Node.js, ASPNET)
+- Download waves to check project accessibilty
+
+### Setup
+
+1. Clone the repo by running the command
+   ```sh
+    git clone git@github.com:umair-rao/COVID-19-Live-Data-Analysis.git
+   ```
+
+### Install
+
+Install this project with:
+
+2. Open the directory of the project
+   ```sh
+   cd COVID-19-Live-Data-Analysis
+
+   ```
+3. Open the directory of the src
+   ```sh
+   cd src
+   ```
+4. Open the index.html file
+   ```sh
+   Click and open the index.html file
+   ```
+
+### Usage
+
+To run the project, execute the following command:
+
+5. Please run the following to render on the browser
+   ```sh
+   npm start
+   ```
+
+### Run tests
+
+To run tests, run the following command:
+
+6. Please run the following to test
+   ```sh
+   npm run test
+   ```
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+7. Please run the following to deploy
+   ```sh
+   npm run deploy
+   ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+
+👤 **Muhammad Umair Muqeem**
+
+- GitHub: [@umair-rao](https://github.com/umair-rao)
+- Twitter: [@umair_ra0](https://twitter.com/umair_ra0)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/engr-umair-muqeem/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Feature <a name="future-features"></a>
+
+- Live COVID data is avaliable of all countires of world.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+- Contributions, issues, and feature requests are welcome!
+
+- Feel free to check the [issues page](https://github.com/umair-rao/COVID-19-Live-Data-Analysis/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+- Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Microverse for providing the Design and guideline for this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] It will be good to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio)[Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the original design. This is a requirement for this project to be approved

Reply:
Readme updated and credit has been given to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio).